src/components/invoiceForm.vue

```diff
@@ -20,7 +20,13 @@
        <h3>ข้อความเพิ่มเติม</h3>
        หัวข้อ : <input v-model="includeNote.subject"/><br>
        เนื้อหา : <textarea v-model="includeNote.content"/><br>
+       <!--Seller Trade Party-->
+       <h3> ข้อมูลผู้ขาย </h3>
+       รหัสผู้ขาย : <input v-model="sellerTradeParty.id"/><br>
+       รหัสผู้ค่าสากล : <input v-model="sellerTradeParty.globalID"/><br>
+       ชื่อผู้ขาย : <input v-model="sellerTradeParty.name"/><br>
        <button @click="buttonIsSubmitted=true"> ส่งข้อมูล </button>
+
      </div>
      <div v-else>
```

Comment:
This if-else branching could be a nice opportunity to split these long forms into two distinct components.

If I were to do this, I would separate them into two components: `InvoiceForm` and `InvoiceConfirmation`.